raiden/ui/cli.py

```diff
@@ -283,10 +291,20 @@ def run(ctx, **kwargs):
 
         app_.raiden.register_registry(app_.raiden.chain.default_registry.address)
 
+        domain_list = []
+        if kwargs['rpccorsdomain']:
+            if ',' in kwargs['rpccorsdomain']:
+                for domain in kwargs['rpccorsdomain'].split(','):
+                    domain = str("".join(["http://", domain, ":*/*"]))
```

Comment:
Looks fine. Only question I have is: why did you choose to not simply give the option for the user to specify the domain fully via the arguments and instead are prepending and appending the http prefix and port/directory to each domain manually here?